indico/modules/events/registration/controllers/__init__.py

```diff
@@ -46,7 +46,8 @@ def _process(self):
         form = make_registration_form(self.regform, management=self.management, registration=self.registration)()
 
         if form.validate_on_submit():
-            modify_registration(self.registration, form.data, management=self.management)
+            notify_user = request.form.get('send_modification_notification') == 'y'
```

Comment:
The `form` is never passed to the template, how do I render the `notify_modification` field?